python/pyspark/sql/functions.py

```diff
@@ -5956,6 +5978,33 @@ def posexplode_outer(col: "ColumnOrName") -> Column:
     return _invoke_function_over_columns("posexplode_outer", col)
 
 
+def inline_outer(col: "ColumnOrName") -> Column:
+    """
+    Explodes an array of structs into a table.
+    Unlike inline, if the array is null or empty then null is produced for each nested column.
+
+    .. versionadded:: 3.4.0
+
+    Examples
+    --------
+    >>> from pyspark.sql import Row
+    >>> from pyspark.sql.functions import inline_outer
```

Comment:
you can remove this since we're in this package, and assume that users know how to import this.

Reply:
Ok. I saw all the PRs about self contained examples and thought that meant copy and pasting the code should run as is. Looked and some of those examples though and saw it was mostly adding those two sections you mentioned, will do

Reply:
Removed